docs/how-to/manage/configure-unprivileged-workloads.rst

```diff
@@ -0,0 +1,42 @@
+.. _configure_unprivileged_workloads:
+
+Configure user workloads to run unprivileged
+============================================
+
+This guide describes how to configure Charmed Kubeflow (CKF) to run all user-generated workloads without elevated permissions.
+
+---------------------
+Requirements
+---------------------
+
+* A Charmed Kubeflow deployment with version 1.10 or later.
+
+-----------------------
+Enable Istio CNI Plugin
+-----------------------
+
+First, follow :ref:`enable Istio CNI<enable_istio_cni>` to enable the Istio CNI plugin in your CKF deployment. This will ensure that all Istio init containers follow the stricter security standards that are enforced later in this guide.
```

Comment:
suggestion (blocking): this way, Istio's init containers are not run at all, so we could instead say that their role is replaced by Istio's DaemonSet

Reply:
suggestion (non-blocking): why don't we include it in the requirements?